### ISSUE TITLE
go-redis import path changed

### ISSUE THREAD

**Issue by smallfish06** [pinpoint-apm/pinpoint-go-agent#61]
Description
---
https://uptrace.dev/blog/posts/go-redis-v9.html

go-redis joined Redis org, so import path has changed. 
V7, V8 import path remain same

```golang
//before
import "github.com/go-redis/redis/v9"
//after
import "github.com/redis/go-redis/v9"
```

**Comment by smallfish06** [comment]
@dwkang can you check this issue? :)

**Comment by dwkang** [comment]
@smallfish06 Thank you for your contribution. It was released in v1.2.2 version.
